src/ziggurat/init.clj

```diff
@@ -88,18 +93,36 @@
   (mount/stop #'server/server)
   (stop-messaging))
 
+(defn start-batch-consumer [args]
+  (-> (mount/only #{#'thread-pool/executor-thread-pool
+                    #'consumer-driver/consumer-groups})
+      (mount/with-args (:batch-routes args))
+      (mount/start)))
+
+(defn stop-batch-consumer []
+  (-> (mount/only #{#'thread-pool/executor-thread-pool
+                    #'consumer-driver/consumer-groups})
+      (mount/stop)))
+
 (def valid-modes-fns
   {:api-server     {:start-fn start-server :stop-fn stop-server}
    :stream-worker  {:start-fn start-stream :stop-fn stop-stream}
    :worker         {:start-fn start-workers :stop-fn stop-workers}
+   :batch-consumer {:start-fn start-batch-consumer :stop-fn stop-batch-consumer}
```

Comment:
Prefer using consistent naming here. Especially in the user-facing side of things i.e. `:batch-worker`